src/zkscript/merkle_trees/merkle_tree.py

```diff
@@ -0,0 +1,198 @@
+from tx_engine import Script
+
+
+class MerkleTree:
+    """Class representing a Merkle Tree.
```

Comment:
This could be more descriptive re the fact that this is used to construct locking script/unlocking script.

Reply:
fixed!